common/src/main/java/org/apache/celeborn/common/network/util/NettyUtils.java

```diff
@@ -122,14 +124,15 @@ private static PooledByteBufAllocator getSharedPooledByteBufAllocator(
         // each core should have one arena to allocate memory
         _allocator = createPooledByteBufAllocator(true, true, conf.networkAllocatorArenas());
         new NettyMemoryMetrics(
-            _allocator, "shared-pool", conf.networkAllocatorVerboseMetric(), source);
+            _allocator, "shared-pool", conf.networkAllocatorVerboseMetric(), source, null);
       }
       return _allocator;
     }
   }
 
   public static PooledByteBufAllocator getPooledByteBufAllocator(
       TransportConf conf, AbstractSource source, boolean allowCache) {
+    Map<String, String> labels = null;
```

Comment:
why define it in the outer of usage scope?

Reply:
thx, updated